translate/lang/ngram.py

```diff
@@ -131,7 +132,7 @@ def classify(self, text):
         ngram = _NGram(text)
         r = 'guess'
 
-        min = sys.maxint
+        min = six.MAXSIZE
```

Comment:
Note that `six.MAXSIZE` is not a replacement for `sys.maxint` https://pythonhosted.org/six/#six.MAXSIZE After reading the code where `min` is used I suppose this works fine.